.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, synchronize, reopened]
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  quality:
+    name: Check quality
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        command: ['ci:attw', 'ci:publint', 'ci:sherif', 'ci:type', 'test', 'build']
```

Comment:
This change make us check automatically in Pull Request
1. attw: https://github.com/arethetypeswrong/arethetypeswrong.github.io
2. publint: https://github.com/bluwy/publint
3. sherif: https://github.com/QuiiBz/sherif
4. tsc: TypeScript check
5. build: Build All